test/fixtures/head-alias/head-alias.test.js

```diff
@@ -0,0 +1,3 @@
+import { buildFixture } from '../../utils/build'
+
+buildFixture('head-alias')
```

Comment:
Can't we extend another fixture (like with-config)? Adding fixtures makes CI slower